test/e2e/storage/testsuites/multivolume.go

```diff
@@ -506,6 +508,14 @@ func TestConcurrentAccessToSingleVolume(f *framework.Framework, cs clientset.Int
 
 	var seed int64
 	byteLen := 64
+	directIO := false
+	// direct IO is needed for Block-mode PVs
```

Comment:
Can you explain why in the comment?

Reply:
When attaching a block PV to a node, the Linux kernel tries to detect a partition table (reading the first sector(s) of the device). Because of read caching, writes to the device from an other node will not show up until the caches are dropped. Dropping caches is a privileged operation, and can not be done from within a (standard) pod. The alternative is to read with caches disabled, which is what direct I/O does.